pkg/util/pravegacluster.go

```diff
@@ -81,8 +82,43 @@ func ConfigMapNameForSegmentstore(clusterName string) string {
 	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
 }
 
-func StatefulSetNameForSegmentstore(clusterName string) string {
-	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
+//function to check if the version is below 0.7 or not
+func IsVersionBelow07(ver string) bool {
+	if ver == "" {
+		return true
+	}
+	normVer, _ := NormalizeVersion(ver)
+	trimNormVer := normVer[0:3]
```

Comment:
Lets use compareVersions instead of picking fixed number of elements from the normVer Array. That would be more elegant.

Reply:
I need to pick the first 3 characters only as otherwise if I am checking for version after normalization it can be 0.6.0 0r 0.6.1 or 0.6.2 thus it will be a very long condition now I can just check if it's equal to 0.6 and it will cover all the 0.6.x type of cases